src/components/PatronDetail/PatronDetail.js

```diff
@@ -7,49 +7,88 @@ import {
   Col,
   NoValue,
   FormattedDate,
+  KeyValue,
+  Loading,
 } from '@folio/stripes/components';
 import { ProfilePicture } from '@folio/stripes/smart-components';
 
-import { getFullName } from '../../util';
 import css from './PatronDetail.css';
+import { usePatronGroup } from '../../hooks';
 
 const PatronDetail = memo(({ user, isUserProfilePicConfigEnabledForTenant }) => {
+  const { data: patronGroup, isLoading: isPatronGroupLoading } = usePatronGroup(user?.patronGroup);
   const profilePictureLink = user?.personal?.profilePictureLink;
 
-  const getUserValue = () => {
-    return (
-      <Row className={css.marginTop}>
-        <Col xs={3}>
-          <strong>
-            {getFullName(user)}
-          </strong>
+  const renderBorrowerDetails = () => (
+    <div className={`${css.borrowerDetails} ${isUserProfilePicConfigEnabledForTenant ? css.borrowerWhenProfilePicConfigActive : ''}`}>
```

Comment:
You can use `classnames` library to combine classes. I think it looks cleaner that way